website/content/docs/v0.7/Local Platforms/virtualbox.md

```diff
@@ -0,0 +1,168 @@
+---
+title: Virtualbox
+---
+
+In this guide we will create a Kubernetes cluster using Virtualbox.
+
+## Installation
+
+### How to get Virtualbox
+
+Install Virtualbox with your operating system package manager or from the [website](https://www.virtualbox.org/).
+For example, on Ubuntu for x86:
+
+```bash
+apt install virtualbox
+```
+
+### Install talosctl
+
+You can download `talosctl` via
+[github.com/talos-systems/talos/releases](https://github.com/talos-systems/talos/releases)
+
+```bash
+curl https://github.com/talos-systems/talos/releases/download/<version>/talosctl-<platform>-<arch> -L -o talosctl
+```
+
+For example version `v0.7.0` for `linux` platform:
+
+```bash
+curl https://github.com/talos-systems/talos/releases/download/v0.7.0/talosctl-linux-amd64 -L -o talosctl
+sudo cp talosctl /usr/local/bin
+sudo chmod +x /usr/local/bin/talosctl
+```
+
+### Download ISO image
```

Comment:
`image` should be capitalized (heading/title)

Reply:
fixed!